index.html

```diff
@@ -43,7 +43,7 @@ <h1 id="features">Features</h1>
   <div class="row">
     <div class="col-lg-4">
       <h4>Modding API</h4>
-      <p>Modify the game and add new content using the <a href="//www.lua.org/">Lua</a> programming language.</p>
+      <p><a href="http://dev.minetest.net/Intro">Modify the game and add new content</a> using the <a href="//www.lua.org/">Lua</a> programming language.</p>
```

Comment:
I think that the fact that all content (nodes, items and tools) is defined by Lua scripts should be emphasised here.